lib/crypto/sig_verifier.go

```diff
@@ -66,12 +64,12 @@ func (sv *SignatureVerifier) Start() {
 			case <-sv.closeCh:
 				return
 			default:
-				sign := sv.Remove()
-				if sign == nil {
+				sig := sv.Remove()
+				if sig == nil {
 					continue
 				}
-				err := sign.verify()
-				if err != nil {
+				ok, err := sig.VerifyFunc(sig.PubKey, sig.Sign, sig.Msg)
+				if err != nil || !ok {
```

Comment:
Please split this in two. Otherwise we might log a `nil` error if `ok` is false.

Reply:
This is not patched, please patch, thanks!